test/distributed/org/apache/cassandra/distributed/test/ByteBuddyExamplesTest.java

```diff
@@ -115,8 +117,12 @@ static void install(ClassLoader cl, int nodeNumber)
 
         public static ResultMessage.Rows execute(QueryState state, QueryOptions options, long queryStartNanoTime, @SuperCall Callable<ResultMessage.Rows> r) throws Exception
         {
-            count.incrementAndGet();
-            return r.call();
+            Rows res = r.call();
+
+            if (res.result.metadata.names.stream().map(c -> c.toString()).collect(Collectors.toList()).contains("bytebuddy_test_column"))
```

Comment:
Super nit: can use a method reference
```suggestion
            if (res.result.metadata.names.stream().map(ColumnSpecification::toString).collect(Collectors.toList()).contains("bytebuddy_test_column"))
```